draft-ietf-acme-acme.md

```diff
@@ -1759,13 +1766,16 @@ Revocation requests are different from other ACME request in that they can be
 signed either with an account key pair or the key pair in the certificate.
 Before revoking a certificate, the server MUST verify that the key used to sign
 the request is authorized to revoke the certificate.  The server SHOULD consider
-at least the following keys authorized for a given certificate:
+at least the following accounts authorized for a given certificate:
 
-* the public key in the certificate.
+* the account that issued the certificate.
 
-* an account key that is authorized to act for all of the identifier(s) in the
+* an account that holds authorizations for all of the identifiers in the
   certificate.
 
+The server SHOULD also consider a revocation request valid if it is signed with
```

Comment:
There may be other CAs that don't want to consider possession of a private key sufficient for automated revocation. For instance they might want to have people submit a certificate problem report and manually validate it. I wanted to lean on the side of not tying their hands.